app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -379,60 +378,44 @@ export default {
       min-height 6.91rem
```

Comment:
@jbibla Imo we need a set of measures that are consistent throughout the application. Those should probably also be used as variables in Sketch. What do you think?